charts/gateway/values.yaml

```diff
@@ -166,109 +172,238 @@ config:
-# Each bundle that you create as a ConfigMap can not exceed 1MB in size.
+## The folllowing must be included
+  # SSG_ADMIN_USERNAME
+  # SSG_ADMIN_PASSWORD
+  # SSG_CLUSTER_PASSWORD
+## If using a Database
+  # SSG_DATABASE_USER
+  # SSG_DATABASE_PASSWORD
 
-# Bundles that contain sensitive information can be mounted using the Kubernetes CSI Driver
-existingBundle:
-  enabled: false
-  configMaps: []
- # - name: mybundle1
-    # configMap:
-    #   defaultMode: 420
-    #   optional: false
-    #   name: mybundle1
- # - name: mybundle2
-  secrets: []
-  # - name: mysecretbundle1
-  #   csi:
-  #     driver: secrets-store.csi.k8s.io
-  #     readOnly: true
-  #     volumeAttributes:
-  #       secretProviderClass: "secret-provider-class-name"
- # - mysecretbundle2
+# Cluster Password
+clusterPassword: mypassword
 
-# Configure custom hosts
-customHosts:
-  enabled: false
-  hostAliases: []
-  # - hostnames:
-  #   - "dev.ca.com"
-  #   - "dev1.ca.com"
-  #   ip: "0.0.0.0"
-  # - hostnames:
-  #   - "example.ca.com"
-  #   ip: "0.0.0.0"
+# This enables/disables Policy Manager Access and sets the SSG_ADMIN username and password
+# Credentials will be moved to a secret object in the next push..
+management:
+  enabled: true
+  # Enable Restman, if DBbacked this setting will persist until manually deleted via Policy Manager.
+  restman:
+    enabled: false
+  # This is the username/password used for Policy Manager/Gateway Management.
+  username: admin
+  password: mypassword
+  # This optionally loads the Kubernetes Service Account token as a secure password for use in Policy.
+  kubernetes:
+    loadServiceAccountToken: false
+  service:
+    enabled: true
+    type: LoadBalancer
+    ## Load Balancer sources
+    ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    annotations: {}
+      # cloud.google.com/load-balancer-type: "Internal"
+    ports:
+      # - name: https
+      #   internal: 8443
+      #   external: 8443
+      #   protocol: TCP
+      - name: management
+        internal: 9443
+        external: 9443
+        protocol: TCP
+    # Uncomment the following section to enable session affinity
+    # sessionAffinity: ClientIP
+    # sessionAffinityConfig:
+    #   clientIP:
+    #     timeoutSeconds: 10800
 
 service:
   # Service Type, ClusterIP, NodePort, LoadBalancer
   type: LoadBalancer
   ## Load Balancer sources
   ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
-  ##
   # loadBalancerSourceRanges:
   # - 10.10.10.0/24
-
   ## Set the ExternalIPs
-  ##
   # externalIPs:
-
   ## Set the LoadBalancerIP
-  ##
   # loadBalancerIP:
 
   # Update this port list if additional ports need to be exposed
   ports:
     - name: https
       internal: 8443
       external: 8443
+      protocol: TCP
     - name: management
```

Comment:
this management port should be commented out for line 370?
also, if we have a strong opinion about how each port should be used, then maybe just define them right in the template instead? what's the benefit of letting users configure these? it's pretty heavy reliance on user knowledge to configure each port right and not conflict

Reply:
There is a production-values.yaml configuration with recommended values.

This has been left as is in the default values.yaml so that existing installations are not accidentally affected by an upgrade.

Reply:
Discussed with @APIM-KiranVaddadi - the default will be (updated)

servicePorts - 8443
managementPorts - 9443